extensions/wikia/AdEngine/js/MonetizationServiceHelper.js

```diff
@@ -0,0 +1,130 @@
+define('ext.wikia.adEngine.monetizationsServiceHelper', [
+	'jquery',
+	'wikia.geo',
+	'wikia.scriptwriter',
+	'wikia.tracker',
+	'wikia.window',
+], function ($, geo, scriptWriter, tracker, window) {
+	'use strict';
+
+	var isEndOfContent = false,
+		track = tracker.buildTrackingFunction({
+			trackingMethod: 'internal',
+			category: 'monetization-module',
+			geo: geo.getCountryCode()
+		});
+
+	function validateSlot(slotName) {
+		if (slotName === 'below_category' && isEndOfContent) {
+			return false;
+		}
+
+		return true;
+	}
+
+	function injectContent(slot, content, success) {
+		scriptWriter.injectHtml(slot, content, function () {
+			var $container = $('#' + slot).find('.monetization-module');
+
+			trackImpression($container);
+			trackClickEvent($container);
+
+			success();
+		});
+	}
+
+	function addInContentSlot(slot) {
```

Comment:
I would rather have this slot added in extension/wikia/AdEngine/js/slot/monetizationInContent.js just like other dynamically added slots are handled.